### PR TITLE
Improved i18n support

### PR DESCRIPTION
**Pull request by ararog**

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -39,20 +41,20 @@ const Footer = () => {
...
+        <div className={classes.title}>{t('products')}</div>
         <a
           href="https://gov.beefy.finance"
           target="_blank"
           rel="noopener noreferrer"
           className={classes.link}
         >
           <i className={`fas fa-landmark ${classes.linkIcon}`}></i>
-          <span>gov</span>
+          <span>{t('gov')}</span>
```
Comment (superbeefyboy) [inline src/components/Footer/Footer.js:57]: here gov represents a subdomain, so translating it is not a good idea

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -62,7 +64,7 @@ const Footer = () => {
           className={classes.link}
         >
           <i className={`fas fa-vote-yea ${classes.linkIcon}`}></i>
-          <span>vote</span>
+          <span>{t('vote')}</span>
```
Comment (superbeefyboy) [inline src/components/Footer/Footer.js:67]: Same as above, this being a a link to a subdomain, a translation might not be needed

File: src/components/Header/Header.js [hunk 3]
```diff
@@ -106,6 +107,7 @@ const Header = ({ links, isNightMode, setNightMode }) => {
 };
 
 const renderLink = (name, label, icon, classes) => {
+  const { t } = useTranslation();
```
Comment (superbeefyboy) [inline src/components/Header/Header.js:110]: Line 110:17:  React Hook "useTranslation" is called in function "renderLink" which is neither a React function component or a custom React Hook function         react-hooks/rules-of-hooks

File: src/components/Header/Header.js [hunk 4]
```diff
@@ -115,15 +117,16 @@ const renderLink = (name, label, icon, classes) => {
...
     >
       <i className={`fas fa-${icon} ${classes.icon}`} />
-      <span>{label}</span>
+      <span>{t(label)}</span>
     </a>
   );
 };
 
 const renderLinkSidebar = (name, label, icon, classes) => {
+  const { t } = useTranslation();
```
Comment (superbeefyboy) [inline src/components/Header/Header.js:126]: Same as above, Maybe you can pass t as a parameter to the fn

**Review by superbeefyboy — CHANGES_REQUESTED** [review]
Some minor changes are needed:
. The t object from useTranslation() might need to be passed as a param to those fn
. Some of the links are named after the subdomains, translating them might be confusing

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (ararog) [reply src/components/Footer/Footer.js:57]: I'm going to fix them asap and get back to you.

**Review by superbeefyboy — APPROVED** [review]

**Pull request merged**